third_party/amd/python/triton_amd.cc

```diff
@@ -64,6 +66,8 @@ void init_triton_amd_passes_ttgpuir(py::module &&m) {
                      mlir::createTritonAMDGPUStreamPipelinePass);
   ADD_PASS_WRAPPER_1("add_stream_pipelinev2",
                      mlir::createTritonAMDGPUStreamPipelineV2Pass, int);
+  ADD_PASS_WRAPPER_0("add_amdgpu_to_llvm",
+                     mlir::triton::createConvertAMDGPUToLLVMPass);
```

Comment:
why do we need a separate pass for  AMDGPU to LLVM? Ideally this is done with extra pattern on top of TritonGPU to llvm as breaking up the conversion to llvm into multiple pass has downsides. It means we get an intermediate IR that mixed block and simt semantic and it prevents pattern matching when lowering.